reduct-nepal/seep
# added a confirmation button in delete audio section

## Completed

- [x] Adding a simple confirm delete dialog box to make sure super user really intended to delete the audio.
## Testing
To test:
- Login into Reduct Nepal Testing Portal as SuperAdmin
- Try deleting a audio in Audio Section
- You should be prompted with a dialog box for confirmation.
